pkgs/tools/networking/rpki-arin-tal/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib, stdenv, fetchurl, writeScript, bash }:
+
+stdenv.mkDerivation rec {
+  pname = "rpki-arin-tal";
+  version = "1.0";
+
+  src = fetchurl {
+    url = "https://www.arin.net/resources/manage/rpki/arin.tal";
+    hash = "sha256-T2weRW/lq0aL6sFJXlfZmh7uqk2fnjRRnq9YhXwhr0g=";
+  };
+
+  builder = writeScript "builder.sh" ''
+    source $stdenv/setup
+    mkdir -p $out
+    cp $src $out/arin.tal
+  '';
+
+  meta = with lib; {
+    description = ''
+       ARIN's RPKI Trust Anchor Locator (TAL)
+       With downloading this package you aggree to the ARIN RPA 
+       https://www.arin.net/resources/manage/rpki/rpa_092622.pdf
+
+       Context: https://mailman.nanog.org/pipermail/nanog/2022-October/220786.html
```

Comment:
This must be moved to the license.

Reply:
No, this is state the software works with, but doesn't have to. This does not affect the software's license.